### DISCUSSION TITLE
WebDav挂载百度网盘，无法上传文件，提示“你需要权限来执行此操作”。

### DISCUSSION THREAD

**Discussion by jerry945** [AlistGo/alist#1639] [Q&A] [answered]
RaiDrive挂载百度网盘，可以下载文件，不能上传文件。往挂载目录丢文件提示“提示“你需要权限来执行此操作”。

google 网盘可以上传。

**Comment by tom200509** [comment]
我也遇到了这种情况，你解决了问题吗？

**Reply by tom200509** [reply]
我目前的解决方案是：改为使用alist2.6.4版本。

**Reply by iMsazy** [reply]
高版本的不可以吗？

**Comment by aday1986** [comment]
我也遇到了相同的我问题。

**Comment by CircusChen** [comment]
同问 一样的问题

**Comment by lujaton** [comment]
为什么v3不行？

**Comment by vintue** [comment]
新来的遇到同样的问题

**Comment by fordiy** [comment]
对，无论3.6版的alist还是3.8版的，通过RaiDarive映射后都不行，都说要权限！

**Comment by anwen-anyi** [comment, marked as answer]
完美回答 记得标记一下~
https://alist.nn.ci/zh/faq/why.html#raidrive挂载百度网盘-可以下载文件-不能上传文件。往挂载目录丢文件提示-提示-你需要权限来执行此操作-。这个怎么解决

**Reply by Zhaohhya** [reply]
回答很好，但没有解决

**Reply by anwen-anyi** [reply]
> 回答很好，但没有解决
> 
> 

就这样解决不了 要么你回去用V2的

**Comment by Cazaea** [comment]
同样问题。

**Comment by spiritedRunning** [comment]
如果说v2版本会忽略空文件上传， 那最新版本为什么不能加上这个，是有什么其他问题吗？ 不然这问题不是无解了

**Comment by heli9913** [comment]
RaiDrive挂载百度网盘，可以下载文件，不能上传文件。往挂载目录丢文件提示“提示“你需要权限来执行此操作”。这个怎么解决？
Raidrive 是因为上传文件会预先创建一个空文件 而百度网盘不允许创建空文件。 v2可以成功是因为v2会忽略空文件上传

**Comment by bihvdgj** [comment]
115网盘挂载了也上传不了。怎么解决。大佬们

**Comment by Mdgas** [comment]
阿里网盘也是不行，有解决办法吗？

**Comment by heli9913** [comment]
尝试选择代理+302
----- 原始邮件 -----
发件人：Mdgas ***@***.***>
收件人：alist-org/alist ***@***.***>
抄送人：heli9913 ***@***.***>,  Comment ***@***.***>
主题：Re:_[alist-org/alist]_WebDav挂载百度网盘，无法上传文件，提示“你需要权限来执行此操作”。_(Discussion_#1639)
日期：2023年03月15日 20点58分


阿里网盘也是不行，有解决办法吗？

&mdash;

**Comment by qq392784506** [comment]
用airlivedrive，解决

**Reply by montezl** [reply]
远程服务器返回错误: (405) 不允许的方法。

**Comment by ZhANGJNG** [comment]
挂载后将文件夹属性的【只读】选项取消勾选，可以上传文件
